crs-setup.conf.example

```diff
@@ -622,20 +622,49 @@ SecDefaultAction "phase:2,log,auditlog,pass"
 #
 # To use geolocation, we make use of the MaxMind GeoIP database.
 # This database is not included with the CRS and must be downloaded.
-# You should also update the database regularly, for instance every month.
-# The CRS contains a tool to download it to util/geo-location/GeoIP.dat:
-#   util/upgrade.py --geoip
 #
-# This product includes GeoLite data created by MaxMind, available from:
-# http://www.maxmind.com.
+# There are two formats for the GeoIP database. ModSecurity v2 uses v1 (.dat files),
+# and ModSecurity v3 uses v2 (.mmdb files).
```

Comment:
Instead of v1 and v2 can we say GeoLite and GeoLite2 here?

Reply:
Yes, it is better that way. Otherwise in confusing.